pmd-java/src/test/java/net/sourceforge/pmd/cli/CLITest.java

```diff
@@ -46,6 +52,14 @@ public void changeJavaVersion() {
                 FileUtil.findPatternInFile(new File(resultFilename), "Using Java version: Java 1.5"));
     }
 
+    @Test
+    public void changeJavaVersionLongOption() {
+        String[] args = { "-d", SOURCE_FOLDER, "-f", "text", "-R", "category/java/design.xml", "--use-version", "1.5", "--language", "java", "-debug", };
```

Comment:
This is not how I'd like `--use-version` to be used. It should read 
```suggestion
        String[] args = { "-d", SOURCE_FOLDER, "-f", "text", "-R", "category/java/design.xml", "--use-version", "java", "1.5", "-debug", };
```